analysis/wwz/make_datacards.py

```diff
@@ -498,8 +516,11 @@ def main():
     # Get the syst ratios to nominal (i.e. kappas)
     kappa_dict = None
     if do_nuis:
-        kappa_dict = get_kappa_dict(yld_dict_mc,yld_dict_data)
-        kappa_dict = add_stats_kappas(yld_dict_mc,kappa_dict,skip_procs=["ZZ","ttZ"])
+        kappa_dict = get_kappa_dict(yld_dict_mc,yld_dict_data,yrs_lst)
+        # Don't do mc stats kappas for data-driven bkg if doing TFs
+        if do_tf: skip_stats_kappas_lst = ["ZZ","ttZ"]
```

Comment:
Just a thought, should we make a print statement explicitly saying that transfer factor option is being used (since in theory we don't want to be doing this)?

Reply:
We could, though there are already so many other print statements that the data card maker prints that I think it would not really be noticed. 

The default behavior is updated though, so that it's off by default (you have to explicitly include it as a command line option to turn it on).

I've also added a wrapper sh script with the "official" way to run the datacard maker (with the settings we want for the analysis) so that will also hopefully help to mitigate potential danger of randomly turning it on.

Reply:
I see, I think that is a good idea and I like the wrapper script a lot